src/main/scala/intellij/haskell/editor/HaskellCompletionContributor.scala

```diff
@@ -182,6 +184,10 @@ class HaskellCompletionContributor extends CompletionContributor {
                 resultSet.addAllElements(findTopLevelTypeSignatureLookupElements(psiFile).asJavaCollection)
               }
 
+              if (HaskellConsoleView.isConsole(psiFile)) {
```

Comment:
The normal completion contributer can not work as expected, that' why I add that piece of code to add special result set for console file😄